### DISCUSSION TITLE
Messages to self are duplicated

### DISCUSSION THREAD

**Discussion by Florian7843** [AsamK/signal-cli#618] [Q&A] [answered]
Hey, I did some testing with your awesome CLI, and wanted to use it to automatically send messages to myself. 

One thing that bothers me is that messages I direct to my own phone number are shown twice on my phone.

Is this intended behavior/The fault of Signal itself or is there a fix to this?

**Comment by AsamK** [comment, marked as answer]
That's the current behavior of libsignal-service-java.
To send note to self messages, as the apps do, you can use `send --note-to-self`

**Reply by Florian7843** [reply]
Works perfectly. Thanks for the quick reply.
